aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/Api/IdeasController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq;
+using Microsoft.AspNet.Mvc;
+using TestingControllersSample.ClientModels;
+using TestingControllersSample.Core.Interfaces;
+using TestingControllersSample.Core.Model;
+
+namespace TestingControllersSample.Api
+{
+    [Route("api/ideas")]
+    public class IdeasController : Controller
+    {
+        private readonly IBrainstormSessionRepository _sessionRepository;
+
+        public IdeasController(IBrainstormSessionRepository sessionRepository)
+        {
+            _sessionRepository = sessionRepository;
+        }
+
+        [Route("forsession/{sessionId}")]
+        [HttpGet]
+        public IActionResult ForSession(int sessionId)
+        {
+            var session = _sessionRepository.GetById(sessionId);
+            if (session == null)
+            {
+                return HttpNotFound(sessionId);
+            }
+            return new ObjectResult(session.Ideas.Select(i => new IdeaDTO()
+            {
+                id = i.Id,
+                name = i.Name,
+                description = i.Description,
+                dateCreated = i.DateCreated
+            }));
+        }
+
+        [Route("create")]
+        [HttpPost]
+        public IActionResult Create([FromBody]NewIdeaModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return new BadRequestObjectResult(ModelState);
+            }
+            var session = _sessionRepository.GetById(model.SessionId);
+            if (session == null)
+            {
+                return HttpNotFound(model.SessionId);
+            }
+            var idea = new Idea()
+            {
+                DateCreated = DateTime.Now,
```

Comment:
`DateCreated` is of type `DateTimeOffset`, so this should be `DateTimeOffset.Now`?